From 519177b73d6c7a5f6d7a9c037946672159bcdd35 Mon Sep 17 00:00:00 2001
From: "tong.gao" <tong.gao@amlogic.com>
Date: Mon, 13 Nov 2023 14:56:22 +0800
Subject: [PATCH 07/14] transcoding: build amlvconv [1/2]

PD#SWPL-145366

Problem:
add transcoding

Solution:
add .bb conf

Verify:
local

Signed-off-by: tong.gao <tong.gao@amlogic.com>
Change-Id: Ic42124155864e8bd0b8714191692c901252e53dd
---
 conf/machine/mesonsc2-ah212-weston-5.15.config      | 1 +
 recipes-multimedia/gst-aml-utils/gst-aml-gfx2d.bb   | 4 ++--
 recipes-multimedia/gst-aml-utils/gst-plugin-conv.bb | 2 +-
 3 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/conf/machine/mesonsc2-ah212-weston-5.15.config b/conf/machine/mesonsc2-ah212-weston-5.15.config
index 5aa94088..798ef7b9 100644
--- a/conf/machine/mesonsc2-ah212-weston-5.15.config
+++ b/conf/machine/mesonsc2-ah212-weston-5.15.config
@@ -6,6 +6,7 @@
 include conf/machine/include/mesonsc2_k5.15_64b.inc
 include conf/machine/include/amlogic_weston.inc
 include conf/machine/include/amlogic_aiot_common.inc
+include conf/machine/include/amlogic_aiot_nndemo.inc
 
 PREFERRED_PROVIDER_virtual/mesa = "libgles-eabihf-dvalin-wayland-drm"
 PREFERRED_PROVIDER_virtual/mesa-gl = "libgles-eabihf-dvalin-wayland-drm"
diff --git a/recipes-multimedia/gst-aml-utils/gst-aml-gfx2d.bb b/recipes-multimedia/gst-aml-utils/gst-aml-gfx2d.bb
index ce78db1d..3afe8301 100755
--- a/recipes-multimedia/gst-aml-utils/gst-aml-gfx2d.bb
+++ b/recipes-multimedia/gst-aml-utils/gst-aml-gfx2d.bb
@@ -4,8 +4,8 @@ LICENSE = "AMLOGIC"
 LIC_FILES_CHKSUM = "file://${COREBASE}/../meta-meson/license/AMLOGIC;md5=6c70138441c57c9e1edb9fde685bd3c8"
 
 DEPENDS += " gstreamer1.0 gstreamer1.0-plugins-base"
-DEPENDS += " libge2d"
-RDEPENDS_${PN} += " libge2d"
+DEPENDS += " libge2d dma-allocator"
+RDEPENDS_${PN} += " libge2d dma-allocator"
 
 inherit autotools pkgconfig
 
diff --git a/recipes-multimedia/gst-aml-utils/gst-plugin-conv.bb b/recipes-multimedia/gst-aml-utils/gst-plugin-conv.bb
index f1787f79..d1afa828 100644
--- a/recipes-multimedia/gst-aml-utils/gst-plugin-conv.bb
+++ b/recipes-multimedia/gst-aml-utils/gst-plugin-conv.bb
@@ -4,7 +4,7 @@ LICENSE = "AMLOGIC"
 LIC_FILES_CHKSUM = "file://${COREBASE}/../meta-meson/license/AMLOGIC;md5=6c70138441c57c9e1edb9fde685bd3c8"
 
 DEPENDS = " gstreamer1.0 gstreamer1.0-plugins-base"
-DEPENDS += " gst-aml-dma-allocator gst-aml-gfx2d"
+DEPENDS += " gst-aml-dma-allocator gst-aml-gfx2d gst-aml-drmbufferpool-plugins"
 RDEPENDS_${PN} += " dma-allocator gst-aml-dma-allocator gst-aml-gfx2d libge2d"
 
 #For common patches
-- 
2.25.1

